src/main/java/com/wavefront/AbstractTraceLoader.java

```diff
@@ -61,6 +65,8 @@ public void start(String[] args) {
     } catch (Throwable t) {
       handleProgramExit(t);
     }
+    System.out.println("\u001B[33m" + String.format("Max heap memory: %.2f GB", maxHeapMemory) );
+    System.out.println("\u001B[33m" + String.format("Used heap memory: %.2f GB", usedHeapMemory));
```

Comment:
I prefer to have 1 string instead of 2. Something like this 
"1.5GB memory used from 2.0GB total."
or more simple:
"Memory usage - 1.5GB/2.0GB"